kws/bin/test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2021 Binbin Zhang(binbzha@qq.com)
```

Comment:
this code is original from score.py, so I don't change the copyright infomation

Reply:
better to name this file to compute_accuracy.py, test.py is ambiguous.